canvas_course_site_wizard/controller.py

```diff
@@ -299,12 +262,51 @@ def create_canvas_course(sis_course_id, sis_user_id, bulk_job=None):
 
     # if this creation is part of a single course creation process return
     # the course along with the new job_id. The start_course_template_copy method will updated
-    # the worng record if job id is no supplied.
+    # the wrong record if job id is no supplied.
     if course_job_id:
         return new_course, course_job_id
 
     return new_course
 
+def get_or_create_account(course_data, sis_course_id, course_job_id, bulk_job_id):
```

Comment:
I think that the method name `get_or_create_account` implies that it will return an account; it's not needed here, but it could trip someone up in the future if they want to call the method from somewhere else.

Reply:
Updated and now this function returns the account 🙂